backend/graph/api/model/episode.go

```diff
@@ -64,9 +64,15 @@ func EpisodeFrom(ctx context.Context, e *common.Episode) *Episode {
 		episodeType = EpisodeTypeStandalone
 	}
 
+	status := StatusPublished
+	if e.Unlisted() {
+		status = StatusUnlisted
+	}
```

Comment:
This was very confusing, because there is a common.Status and the model.Status. They have the same name for the type and the constants.

At first, I was thinking: "Why cant you just use the status directly"? And what about the other values?
I understand what's going on, but it's confusing.

It would help with a mapping function like: `status := statusFrom(e.Status)`

Reply:
The GQL API is just one of the interfaces that we create from the common models, and in some cases the bg-worker might use the data there differently than what the GQL API does.

The other values are irrelevant for the API at the moment, so I see no reason exposing them. Just fallback to published if they do not serve a specific purpose.